Bottom: 4296eadb5dcdf44d521e0ed181aae3bfbf42191f
Top:    d318e346c900410ed1f66042b175806979eb6fe4
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-10-01 11:23:03 +0100

Fix outline drawing to only apply on copper layers

This matches comments in the code, and fixes (removes) outline drawing
on the solder mask prints.

This partially reverts commit 294494e56695251e63c0da9f243d6f59447f1c9f

"Fix PS rect/outline logic"


---

diff --git a/src/hid/ps/ps.c b/src/hid/ps/ps.c
index 7f2ae8b..1f25448 100644
--- a/src/hid/ps/ps.c
+++ b/src/hid/ps/ps.c
@@ -1001,10 +1001,11 @@ ps_set_layer (const char *name, int group, int empty)
      we want to "print outlines", and we have an outline layer,
      print the outline layer on this layer also.  */
   if (global.outline &&
+      global.is_copper &&
       global.outline_layer != NULL &&
       global.outline_layer != PCB->Data->Layer+idx &&
-      strcmp (name, "outline") != 0 &&
-      strcmp (name, "route") != 0
+      strcmp (name, "outline") &&
+      strcmp (name, "route")
       )
     {
       dapi->draw_layer (global.outline_layer, &global.region, NULL);
